src/menus/MenuHelper.cpp

```diff
@@ -0,0 +1,782 @@
+#include "MenuHelper.h"
+#include "PluginManager.h"
+#include "effects/EffectManager.h"
+#include "CommonCommandFlags.h"
+#include "prefs/EffectsPrefs.h"
+#include "BatchCommands.h"
+#include "BatchProcessDialog.h"
```

Comment:
Do all these #include directives compile on all platforms?  I thought ../ would be needed in some of them.

Reply:
I believe so, yes. If I remember correctly `src` folder is added to include directories somewhere in cmake